cmd/kubeadm/app/cmd/config.go

```diff
@@ -214,9 +215,59 @@ func NewCmdConfigImages(out io.Writer) *cobra.Command {
 		RunE:  cmdutil.SubCmdRunE("images"),
 	}
 	cmd.AddCommand(NewCmdConfigImagesList(out))
+	cmd.AddCommand(NewCmdConfigImagesPull())
 	return cmd
 }
 
+// NewCmdConfigImagesPull returns the `config images pull` command
+func NewCmdConfigImagesPull() *cobra.Command {
+	cfg := kubeadmapiv1alpha1.MasterConfiguration{}
```

Comment:
must use `&` pointer here

Reply:
It's _technically_ not a must, but practically. If you init it as a pointer, you don't have to remember to pass `&` all the time. I'm pretty sure most things break if someone somehow forgets to pass the pointer at some time. Would be super hard to debug and notice.

Reply:
fair point 👍 thanks for explaining what you meant